server/odc-service/src/main/java/com/oceanbase/odc/service/datatransfer/loader/PlSqlFormat.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.datatransfer.loader;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import com.oceanbase.odc.common.util.StringUtils;
+
+/**
+ * @author youshu
+ * @date 2024/8/27
+ */
+public class PlSqlFormat {
```

Comment:
PLSqlDeveloperExportFormat ?

1. it is the file format used by PL/SQL Developer tool exported, not PL/SQL languange identified.
2. Sql should be SQL better indeed, however too much Sql used in odc already..

Reply:
OK